x-pack/legacy/plugins/index_lifecycle_management/public/sections/edit_policy/components/hot_phase/hot_phase.js

```diff
@@ -254,6 +278,45 @@ export class HotPhase extends PureComponent {
                             defaultMessage: 'hours',
                           }),
                         },
+                        {
+                          value: 'm',
+                          text: i18n.translate('xpack.indexLifecycleMgmt.hotPhase.minutesLabel', {
+                            defaultMessage: 'minutes',
+                          }),
+                        },
+                        {
+                          value: 's',
+                          text: i18n.translate('xpack.indexLifecycleMgmt.hotPhase.secondsLabel', {
+                            defaultMessage: 'seconds',
+                          }),
+                        },
+                        {
+                          value: 'ms',
+                          text: i18n.translate(
+                            'xpack.indexLifecycleMgmt.hotPhase.millisecondsLabel',
+                            {
+                              defaultMessage: 'milliseconds',
+                            }
+                          ),
+                        },
+                        {
+                          value: 'microseconds',
```

Comment:
as mentioned in my previous review, i believe the `value` should be `micros`

Reply:
@alisonelizabeth Sorry, I've missed this. I've updated the values.